### DISCUSSION TITLE
Create Separate Launch Command for Test Branch

### DISCUSSION THREAD

**Discussion by Carterpersall** [ChrisTitusTech/winutil#41] [Ideas]
It would be cool if there was a separate command for running the test branch of the script, for both easier testing in a VM and for those who want to use the bleeding-edge script. It could also help find issues before the test branch is pushed into the main branch as more people opt to run it.

**Comment by DeveloperDurp** [comment]
Technically you could run the following command, but is much harder to remember. 
iwr -useb https://raw.githubusercontent.com/ChrisTitusTech/winutil/test/winutil.ps1 | iex

as a side note I did implement in the test.ps1 a check for the $ENV:environment variable and if it was set to "dev" it would load everything locally. Wouldn't be hard to implement that in the main script at some point that says if $ENV:environment = "test" it would load the test branch files. That way it's still the same command but you could do 
$env:branch = "test" ; iwr -useb https://christitus.com/win | iex

**Comment by ChrisTitusTech** [comment]
I'll create a new alias as I was just running into this today. The big issue is passing the XML from the test branch as the ps1 file will pull from the main branch.
